toc.js

```diff
@@ -139,3 +139,18 @@ window.addEventListener( 'scroll', function () {
 }, false );
 
 }( document, window ) );
+
+document.addEventListener('DOMContentLoaded', function() {
+    var $menu = document.getElementById('nav');
+    var $humberger = document.getElementById("humberger-menu");
+    var $checkbox = document.querySelector("#humberger-menu input");
+
+    $checkbox.addEventListener('change', function(event) {
+        var checkbox = event.target;
+        if(checkbox.checked) {
+            $menu.classList.add('activate');
+        } else {
+            $menu.classList.remove('activate');
+        }
+    });
+})
```

Comment:
This is not generating the table of contents, so should not be in `toc.js`; it should be its own script. The code style should match the code style used elsewhere.

Reply:
Done